components/org.wso2.carbon.identity.data.publisher.oauth/src/main/java/org/wso2/carbon/identity/data/publisher/oauth/listener/OAuthTokenIssuanceLogPublisher.java

```diff
@@ -245,9 +265,119 @@ private void addStringToMap(String name, String value, Map<String, Object> param
         }
     }
 
-    private void logTransactionInfo(String info) {
+    /**
+     * printing the log for given log info type
```

Comment:
Start with the capital letter and end with the full-stop.

Reply:
fixed with https://github.com/wso2-extensions/identity-data-publisher-oauth/pull/74